plotutil/xyer.go

```diff
@@ -0,0 +1,77 @@
+package plotutil
```

Comment:
Please make sure there is a blank line between the license comment and the `package` declaration statement, or else godoc will think that the license is a doc comment.